### PR TITLE
Demo refresh

### PR DESCRIPTION
**Pull request by zimmerel**
Here are my changes for refreshing the look of the demo page.

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-  <div class="logs">
-    <ng-container *ngFor="let entry of logEntries">
-      <span class="log-entry">{{entry}}</span>
-    </ng-container>
+  <img class="logo" src="./assets/bugsplat-logo.png">
+  <div class="content">
+    <h1>
+      Welcome to {{ title }}!
+    </h1>
+    <p>This is a sample application that demonstrates <a [href]="links.home.href">{{ links.home.text }}</a> error reporting for <a [href]="links.angular.href">{{ links.angular.text }}</a> applications build with JavaScript or <a [href]="links.typescript.href">{{ links.typescript.text }}.</a></p>
```
Comment (bobbyg603) [inline src/app/app.component.html:7]:
```suggestion
    <p>This is a sample application that demonstrates <a [href]="links.home.href">{{ links.home.text }}</a> error reporting for <a [href]="links.angular.href">{{ links.angular.text }}</a> applications built with JavaScript or <a [href]="links.typescript.href">{{ links.typescript.text }}.</a></p>
```

File: src/app/app.component.ts [hunk 2]
```diff
@@ -6,50 +6,74 @@ import { MyAngularErrorHandler } from './my-angular-error-handler';
...
+      href: 'https://docs.bugsplat.com/introduction/getting-started/integrations/web/angular',
+      text: 'Angular',
+    },
+    typescript: {
+      href: 'https://docs.bugsplat.com/introduction/getting-started/integrations/web/angular#source-maps',
+      text: 'TypeScript',
+    },
+  };
+
+  errors = [
```
Comment (bobbyg603) [inline src/app/app.component.ts:31]: This is slick, well done!

**Review by bobbyg603 — APPROVED** [review]

**Comment by bobbyg603** [comment]
Hey, @splatyourbugs this looks good to me! Can you add your review, please? Thanks!

**Comment by splatyourbugs** [comment]
@bobbyg603 of course! @zimmerel my feedback is below.

**Comment by splatyourbugs** [comment]
Initial feedback on Demo-refresh: 

1. Shrink size of the BugSplat logo to about half it's current size 
2. Replace background pattern with this pattern (https://www.dropbox.com/s/b9aiiwqdk96d4gg/background-option-5.png?dl=0)
3. Pattern the background asset 
4. Apply slight opacity to the background asset 

Mocked image of changes - 

review-angular-tester-page

**Comment by bobbyg603** [comment]
Build was broken by https://stackoverflow.com/questions/69042794/karma-fails-to-capture-chrome-v93-and-times-out-gives-up# but should be fixed for next check in

**Review by splatyourbugs — APPROVED** [review]
This looks great! 👏

**Pull request merged**